foreman/data_refinery_foreman/surveyor/array_express.py

```diff
@@ -460,11 +460,6 @@ def create_samples_from_api(self,
                 original_file_sample_association.sample = sample_object
                 original_file_sample_association.save()
 
-            association = ExperimentSampleAssociation()
```

Comment:
Can you explain this removal? It isn't mentioned in the PR text.

Reply:
It was duplicated immediately below, except with in a try block.